src/selectors/appSettings.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import { useSelector } from 'react-redux';
+import type { RootReducerState } from 'reducers/rootReducer';
+import type { AppSettingsReducerState } from 'reducers/appSettingsReducer';
+
+export const useAppSettingsSelector = <T>(selector: (state: AppSettingsReducerState) => T): T => {
+  return useSelector((root: RootReducerState) => selector(root.appSettings));
+};
```

Comment:
Do we need the `return` here? 

```suggestion
export const useAppSettingsSelector = <T>(selector: (state: AppSettingsReducerState) => T): T =>
  useSelector((root: RootReducerState) => selector(root.appSettings));
```

Reply:
Actually, my VS Code has trouble correctly coloring code without return because of line length or something.

<img width="969" alt="Screenshot 2021-02-17 at 18 08 40" src="https://user-images.githubusercontent.com/6368606/108240515-34f6af80-714b-11eb-909a-b3785d0fb352.png">

Reply:
Fixed anyway